### DISCUSSION TITLE
How to change the password inside the admin panel?

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#5392] [Archive]
A user should be able to change his/her password. But it does not work within `/admin/users/`. I’m always getting an error like:

```
{
  "message": "An unexpected error occurred.",
  "extensions": {
    "code": "INTERNAL_SERVER_ERROR"
  }
}
```

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/issues/5310 👍🏻
